script/DeployERC721ReceiveTokenWithPredeployedSeadropRedeemToken.s.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {Script} from "forge-std/Script.sol";
+import {Test} from "forge-std/Test.sol";
+import {ItemType} from "seaport-types/src/lib/ConsiderationEnums.sol";
+import {OfferItem, ConsiderationItem} from "seaport-types/src/lib/ConsiderationStructs.sol";
+import {CampaignParams, CampaignRequirements} from "../src/lib/RedeemablesStructs.sol";
+import {ERC721RedemptionMintable} from "../src/extensions/ERC721RedemptionMintable.sol";
+import {ERC721OwnerMintable} from "../src/test/ERC721OwnerMintable.sol";
+import {ERC721ShipyardRedeemableMintable} from "../src/extensions/ERC721ShipyardRedeemableMintable.sol";
+
+contract DeployERC721ReceiveTokenWithPredeployedSeadropRedeemToken is Script, Test {
+    address constant _BURN_ADDRESS = 0x000000000000000000000000000000000000dEaD;
```

Comment:
let's just make this a constant in a new file `src/lib/RedeemablesConstants.sol` and import it across all our files we use it :)